### PR TITLE
Build the doc-json target as part of the install target. Also:

### PR DESCRIPTION
**Pull request by kc284**
Restructured output so it's easier to drop it into xapi-project.github.io.
Added missing release date (or the release is not generated by gen-json.exe).

### CODE REVIEW

**Review by psafont — APPROVED** [review]

File: ocaml/idl/datamodel_types.ml [hunk 1]
```diff
@@ -275,7 +275,7 @@ let release_order_full = [{
      version_major = 2;
      version_minor = 15;
      branding      = "Citrix Hypervisor 8.2 Hotfix 2";
```
Comment (edwintorok) [inline ocaml/idl/datamodel_types.ml:277]: Should we add a 7.1 CU2 to this list? CU1 is there, but CU2 seems to have ben forgotten

**Review by edwintorok — APPROVED** [review]

File: ocaml/idl/datamodel_types.ml (hunk 1 above)
Comment (kc284) [reply ocaml/idl/datamodel_types.ml:277]: It should probably be the other way round, i.e. CU1 should be removed. There were no public API changes either in 7.1 CU1 or 7.1 CU2 (apart from a few new API errors, which don't have version info). Indeed this page http://xapi-project.github.io/xen-api/releases/honolulu.html is empty, so I don't know why CU1 was defined.

**Comment by kc284** [comment]
Don't merge this yet, I may add a modification pending on the outcome of the conversation on https://github.com/xapi-project/xapi-project.github.io/pull/250

**Comment by kc284** [comment]
Any idea why removal of `rel_honolulu` from the datamodel_types.ml is causing the Ocaml tests failure? I don't see it being used anywhere else, am I missing something?

**Comment by robhoes** [comment]
@kc284 
```
New schema hash ('633a99b46dda090677598aad3a830f76') doesn't match the last known one. Please bump the
datamodel schema versions if necessary, and update 'last_known_schema_hash'.

  schematest alias ocaml/idl/runtest (exit 1)
(cd _build/default/ocaml/idl && ./schematest.exe)
```

**Comment by kc284** [comment]
I haven't changed any fields of API objects in this PR, only removed an unused release from Datamodel_types, I think this does not warrant a schema bump, correct?

**Comment by robhoes** [comment]
Yeah, but still the hash may be different for some reason. So don't bump the schema version, but update the hash for the test to pass.

**Comment by kc284** [comment]
Can this be merged now?

**Review by robhoes — APPROVED** [review]

**Pull request merged**